packages/cloudflare/src/api/overrides/incremental-cache/r2-incremental-cache.ts

```diff
@@ -2,13 +2,13 @@ import { debug, error } from "@opennextjs/aws/adapters/logger.js";
 import type { CacheValue, IncrementalCache, WithLastModified } from "@opennextjs/aws/types/overrides.js";
 import { IgnorableError } from "@opennextjs/aws/utils/error.js";
 
-import { getCloudflareContext } from "./cloudflare-context.js";
+import { getCloudflareContext } from "../../cloudflare-context.js";
 
 /**
- * An instance of the Incremental Cache that uses an R2 bucket (`NEXT_CACHE_R2_BUCKET`) as it's
+ * An instance of the Incremental Cache that uses an R2 bucket (`NEXT_INC_CACHE_R2_BUCKET`) as it's
  * underlying data store.
  *
- * The directory that the cache entries are stored in can be configured with the `NEXT_CACHE_R2_PREFIX`
+ * The directory that the cache entries are stored in can be configured with the `NEXT_INC_CACHE_R2_PREFIX`
  * environment variable, and defaults to `incremental-cache`.
  *
  * The cache uses an instance of the Cache API (`incremental-cache`) to store a local version of the
```

Comment:
just noticed this comment must have been left behind, would you be able to delete it at the same time by any chance?